src/sslsocket.cpp

```diff
@@ -131,6 +131,21 @@ SSL* SSL_new_client()
 
 SSL* SSL_new_server()
 {
+
+    if (SSL_CTX_use_certificate_file(sip_trp_ssl_ctx,
+                                     tls_cert_name,
+                                     SSL_FILETYPE_PEM) != 1) {
+        ERROR("SSL_new_server: SSL_CTX_use_certificate_file failed");
+        return NULL;
+    }
+
+    if (SSL_CTX_use_PrivateKey_file(sip_trp_ssl_ctx,
+                                     tls_key_name,
+                                     SSL_FILETYPE_PEM) != 1) {
```

Comment:
For indentation all the way to the right, please indent exactly to after the `(`, like above.